lib/brainstem/dsl/base_block.rb

```diff
@@ -22,8 +22,23 @@ def descend(klass, new_config = configuration, new_options = {}, &block)
 
         def parse_args(args)
           options = args.last.is_a?(Hash) ? args.pop : {}
-          description = args.shift
-          [description, options]
+          maybe_description = args.shift
+
+          if maybe_description.is_a?(Hash)
+            options = options.merge(maybe_description)
```

Comment:
I'm thinking if the user provided two hashes: 1. for the description as `{ info: ... }` 2. for the options `{ ... }`. Since we're still supporting a third argument, it's possible users will rely to heavily on invalid args checks from ruby.